cockatrice/src/main.cpp

```diff
@@ -97,6 +100,19 @@ bool settingsValid()
         !settingsCache->getPicsPath().isEmpty();
 }
 
+void generateClientID()
```

Comment:
I didn't get a chance to look at this before merge; in general I'd really prefer if this returned a QString without touching settings and then we store it outside the method. As it stands it doesn't really follow SRP and the inclusion of a side effect in the method means that we have to refactor it or work around it for testing